app/uk/gov/hmrc/eoricommoncomponent/frontend/views/are_you_sure_remove_vat.scala.html

```diff
@@ -36,53 +45,57 @@
 
 @heading = @{messages("cds.subscription.vat-details.page-remove-vat-heading")}
 
-@inputRadioGroup(field: String, options: Seq[(String, String)]) = {
+@displayInputRadioGroup(field: String, options: Seq[(String, String)]) = {
 
-@helpers.inputRadioGroup(
+@inputRadioGroup(
     removeVatYesNoAnswer(field),
     options,
     Map.empty,
     None,
+    None,
     '_divClass -> "createForm-group form-group")
```

Comment:
Can we use govuk-form-group instead of form-group

Reply:
yes